src/templates/kit/screens/VotingScreen.js

```diff
@@ -32,13 +32,17 @@ function reduceFields(fields, [field, value]) {
   if (field === 'quorum') {
     return {
       ...fields,
+      // 100% quorum is not possible, but any adjustments necessary should be handled in the
+      // templates themselves
```

Comment:
Just a warn here, correct me if wrong:

If `template themselves` means the contract, it might represent a poor UX, since the template will fail at tx level when it could be prevented with basic onboarding checks that could effectively drive the user on the cause and solution. Specially for example on multiple tx process: A succesful first tx followed by a reverted one because the Voting settings were wrong seems awful

Reply:
> If template themselves means the contract

I meant checking for this in the template's frontend implementation. The template contract will fail, because it cannot initialize the Voting contract with those parameters. Will update the wording.